### ISSUE TITLE
Accessibility issue with aria-labelledby: ARIA attribute unsupported or prohibited

### ISSUE THREAD

**Issue by josenino95** [carpentries/varnish#199]
Without further ado, here is what the SiteImprove accessibility tool says

> An ARIA attribute has been specified that is either not supported or is prohibited, on this type of element.

**Occurrences (2)**
For an example with the [Git lesson - Episode 7](https://swcarpentry.github.io/git-novice/07-github.html), see this [SiteImprove report](https://my2.siteimprove.com/Inspector/1237935/A11Y/Page?PageId=343679523083&impmd=A74FC28FD5FAD95A0E08DD9E49865810&issue=cantTell+failed&conf=a+aa+aria+si#/sia-r18/failed/VmdObjBtTnI=;MzMzZjFlZGM=). Notice how this episode in particular has subsections, triggering two occurrences.

In particular, I see SiteImprove is flagging two instances of the use of `aria-labelledby`. One for the entire Collapsible navigation menu with `id="flush-collapseEleven"`, and another time for the selected episode when it has subsections (with `id="flush-headingcurrent")`.
```
<div id="flush-collapseEleven" class="accordion-collapse show collapse" aria-labelledby="flush-headingEleven" data-bs-parent="#accordionFlush11">
```

```
<div id="flush-collapsecurrent" class="accordion-collapse collapse show" aria-labelledby="flush-headingcurrent" data-bs-parent="#accordionFlushcurrent">
```

In only see one occurrence of this issue in pages of episodes where there are not subsections.

> **Why is this an issue?**
> ARIA attributes provide information about the behavior of interactive elements. For example, they can be used to tell screen-reader users whether a checkbox is checked (aria-checked=true) or unchecked (aria-checked=false).
> Unsupported ARIA attributes will not deliver the intended user experience and may even break the usability of your site.
> WCAG 2.2: [Understanding Success Criterion 4.1.2: Name, Role, Value](https://www.w3.org/WAI/WCAG22/Understanding/name-role-value)
> WCAG 2.2: [Technique ARIA5: Using WAI-ARIA state and property attributes to expose the state of a user interface component](https://www.w3.org/WAI/WCAG22/Techniques/aria/ARIA5)
> Siteimprove Alfa: [Technical documentation for rule SIA-R18](https://alfa.siteimprove.com/rules/sia-r18)
